src/randoop/main/Help.java

```diff
@@ -80,7 +80,7 @@ public boolean handle(String[] args_raw) throws RandoopTextuiException {
       // User wants help on a specific command.
       List<CommandHandler> allHandlers = new ArrayList<CommandHandler>();
       allHandlers.addAll(Main.handlers);
-      allHandlers.addAll(Main.invisibleHandlers);
+      // allHandlers.addAll(Main.invisibleHandlers);
```

Comment:
All code related to invisibleHandlers is commented out.  Should it be removed instead?  (I guess my question is why it's kept around -- what do you plan to do with it later?)